configs/warnings.tslint.json

```diff
@@ -47,6 +47,10 @@
     "no-return-await": {
       "severity": "warning",
       "options": true
+    },
+    "no-void-expression": {
+      "severity": "warning",
+      "options": [true, "ignore-arrow-function-shorthand"]
```

Comment:
Right, that's because I hadn't edited the file in Theia.  Fixed it now.  I also removed the `true` in `options`, since Theia complains it doesn't match the schema (I verified the rule works without it).